whylog/tests/test_regexassistant/__init__.py

```diff
@@ -0,0 +1,74 @@
+from unittest import TestCase
+from nose.plugins.skip import SkipTest
+from whylog.regexassistant import RegexAssistant
+from whylog.teacher import Interval
+
+
+class FrontInput(object):
+    """
+    FrontInput for test purpose only while module Front is not implemented yet.
```

Comment:
this class is a good start for the real implementation - you can put it in the appropriate module already. Having two `FrontInput` classes is dangerous.